src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -1,5 +1,5 @@
 import { PresignedPost } from 'aws-sdk/clients/s3'
-import { assignIn, last, omit } from 'lodash'
+import _, { assignIn, last, omit } from 'lodash'
```

Comment:
nit: import destructured function instead of `_`

Reply:
edited